DynamicActiveRecord.php

```diff
@@ -234,16 +234,7 @@ function ($matches) {
      */
     public function __get($name)
     {
-        $value = null;
-        try {
-            $value = parent::__get($name);
```

Comment:
parent::__get() implements not only attributes but also the propery feature of Object and retrieving relations. This can not be replaced by a simple getAttribute() call.